src/beehive/beehive-pipeline.ts

```diff
@@ -0,0 +1,266 @@
+      input: appSourceArtifact,
+      project: smokeTestsProject,
+      actionName: 'SmokeTests',
+      runOrder: 98,
+    })
+
+    // Approval
+    const appRepoUrl = `https://github.com/${props.appRepoOwner}/${props.appRepoName}`
+    const approvalTopic = new Topic(this, 'ApprovalTopic')
+    const approvalAction = new ManualApprovalAction({
+      actionName: 'Approval',
+      additionalInformation: `A new version of ${appRepoUrl} has been deployed to https://${testURL} and is awaiting your approval. If you approve these changes, they will be deployed to stack https://${prodURL}.\n\n*Commit Message*\n${appSourceAction.variables.commitMessage}\n\nFor more details on the changes, see ${appRepoUrl}/commit/${appSourceAction.variables.commitId}.`,
+      notificationTopic: approvalTopic,
+      runOrder: 99, // This should always be the last action in the stage
+    })
+    if (props.env.slackNotifyStackName !== undefined) {
+      const slackApproval = new SlackApproval(this, 'SlackApproval', {
+        approvalTopic,
+        notifyStackName: props.env.slackNotifyStackName,
+      })
+    }
+
+    // Deploy Production Actions
+
+    // Global variables for production space
+    const prodNamespace = `${props.namespace}-prod`
+    const prodSsmPrefix = 'dec-prod-beehive'
+
+    // Deploy Prod actions
+
+    const deployProd = new CDKPipelineDeploy(this, `${props.namespace}-DeployProd`, {
+      contextEnvName: props.env.name,
+      targetStack: `${prodNamespace}-beehive`,
+      dockerhubCredentialsPath: props.dockerhubCredentialsPath,
+      dependsOnStacks: [],
+      infraSourceArtifact,
+      appSourceArtifact,
+      appBuildCommands: [
+        'npm install -g yarn',
+        'yarn install',
+        'yarn build',
+      ],
+      namespace: prodNamespace,
+      additionalContext: {
+        owner: props.owner,
+        contact: props.contact,
+        networkStack: props.env.networkStackName,
+        domainStack: props.env.domainStackName,
+        createDns: props.env.createDns ? 'true' : 'false',
+        'beehive:hostnamePrefix': prodHostnamePrefix,
+        'beehive:appDirectory': '$CODEBUILD_SRC_DIR_AppCode/build',
+        infraDirectory: '$CODEBUILD_SRC_DIR',
+      },
+    })
+    addPermissions(deployProd, prodNamespace)
+
+    deployProd.project.addToRolePolicy(NamespacedPolicy.route53RecordSet(props.prodFoundationStack.hostedZone.hostedZoneId))
+
+    // Smoke Tests Action
+    const smokeTestsProdProject = new codebuild.PipelineProject(this, 'StaticHostSmokeTestsProd', {
+      buildSpec: codebuild.BuildSpec.fromObject({
+        phases: {
+          build: {
+            commands: [
+              `chmod -R 755 ${props.qaSpecPath}`,
+              `newman run ${props.qaSpecPath} --env-var SiteURL=${prodURL}`,
+            ],
+          },
+        },
+        version: '0.2',
+      }),
+      environment: {
+        buildImage: codebuild.LinuxBuildImage.fromDockerRegistry('postman/newman', {
```

Comment:
Specify a version of the Newman container, or use the DockerhubImage.fromNewman method in src/dockerhub-image.ts

Reply:
Do you have a suggested version number?  When I go to Docker Registry site, there is a postman/newman container and I don't see any versioning options there.

Reply:
```suggestion
        buildImage: codebuild.LinuxBuildImage.fromDockerRegistry('postman/newman:5', {
```

This should work as it matches what is in the DockerhubImage file.

Reply:
Thanks. Change has been made.